### ISSUE TITLE
Additional/Specific Audio Settings

### ISSUE THREAD

**Issue by 4ukai** [Ashleyz4/MC_b1.7.3-MegaMod#8]
_You can categorize the Audio Settings on different stuff like let's say you can do:_
**Juke/Note Blocks**,
**Hostile Mobs**/**Neutral Mobs**,
**Passive Mobs**,
**Weather Sounds**,
**Explosion Sounds**,
**Walking/Breaking Blocks**,
**Water & Lava**...

There is many more options.

**Comment by ScarletLovell** [comment]
Good idea. The passive mobs one is already done, however I will be doing all the other ones too for v0.4
